package.json

```diff
@@ -10,7 +10,8 @@
   },
   "scripts": {
     "build": "gulp",
-    "test": "jest"
+    "test": "jest",
+    "lint": "eslint src/** --fix"
```

Comment:
--fix shouldn't be here. This command supposed to get launched by CI, unintentional changes are not welcome at that stage.

Reply:
I removed --fix